charts/camunda-platform/templates/zeebe-gateway/deployment.yaml

```diff
@@ -85,9 +85,7 @@ spec:
               value: "true"
             - name: ZEEBE_BROKER_GATEWAY_MULTITENANCY_ENABLED
               value: "true"
-            - name: ZEEBE_GATEWAY_SECURITY_AUTHENTICATION_IDENTITY_BASEURL
-              value: {{ include "camundaPlatform.identityURL" . | quote }}
-            - name: ZEEBE_BROKER_GATEWAY_SECURITY_AUTHENTICATION_IDENTITY_BASEURL
+            - name: CAMUNDA_IDENTITY_BASE_URL
```

Comment:
> When using Helm charts, you can enable multi-tenancy globally with the flag `global.multitenancy.enabled`. Visit [the Helm chart configuration](https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/README.md#global-parameters) for additional details. - [source](https://docs.camunda.io/docs/next/self-managed/operational-guides/configure-multi-tenancy/#helm-charts)

@aabouzaid / @koevskinikola How does this work in the helm-charts? Is `CAMUNDA_IDENTITY_BASE_URL` automatically configured when `global.multitenancy.enabled` is set to `true`?

@Ben-Sheppard Interestingly, `CAMUNDA_IDENTITY_BASE_URL` is not mentioned in the [Identity configuration variables](https://docs.camunda.io/docs/next/self-managed/identity/deployment/configuration-variables/#core-configuration).

Reply:
@korthout some of the Identity Spring Boot Starter variables are mentioned [here](https://docs.camunda.io/docs/self-managed/platform-deployment/helm-kubernetes/guides/connect-to-an-oidc-provider/?optionsType=env). 

The `CAMUNDA_IDENTITY_BASE_URL` is not documented, but I derived it from the internal variable name. That's one of the reasons why I requested a review from Ben.

Regarding:
> Did you happen to test this manually?

I tested this only through the automated tests in this repo. I'm still not fully familiar with Helm charts to be sure that what I test locally is correctly set up.

Reply:
I believe it was an oversight to not document it, as you can see [here](https://github.com/camunda-cloud/identity/blob/main/spring-boot-autoconfigure/src/main/java/io/camunda/identity/autoconfigure/IdentityProperties.java#L14) there is a property in the autoconifiguration for it to be set and I would expect the structure you have used here